README.md

```diff
@@ -1 +1,75 @@
-# microverse-leaderboard
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard
+
+> A minimalist Leaderboard app
+
+![screenshot](./img/screenshot.png)
+
+## Getting Started
+
+### Prerequisites
+
+- [Webpack](https://webpack.js.org/)
+
+### Setup
+
+* Clone this repository
+```
+git clone https://github.com/crgc/microverse-leaderboard.git
+```
+* Change into folder
+```
+cd microverse-leaderboard
+```
+* Install Webpack
+```
+npm install webpack webpack-cli --save-dev
+```
+* Install dependencies
+```
+npm install --save lodash
+npm install --save-dev style-loader css-loader
+npm install --save-dev html-webpack-plugin
+npm install --save-dev copy-webpack-plugin
+npm install --save-dev webpack-dev-server
+```
+
+### Run
+* Build
+```
+npm run build
+```
+* Start server
+```
+npm start
+```
+* Open your browser at http://localhost:8080/
+
+## Built With
+
+- HTML5 & CSS3
+- Javascript
+- Webpack
+
+## Live Demo
+[To Do List](https://crgc.github.io/microverse-leaderboard/)
```

Comment:
- [ ] **OPTIONAL**
Since it's not required to provide a live demo link for this project. Please keep in mind that this link is broken and not related to the current project, and this might leave a bad impression on future viewers. Let's work on that 😄 🙏

Reply:
Done.

Reply:
Addressed this!